after/plugin/editor.vim

```diff
@@ -24,12 +24,52 @@ local node_formatters = {
   end,
 }
 
+local json_formatter = {
+  function()
+    return {
+      exe = "prettier",
+      args = {"--stdin-filepath", vim.fn.fnameescape(vim.api.nvim_buf_get_name(0)), "--double-quote"},
+      stdin = true
+    }
+  end
+}
+
+local dart_formatter = {
+    function()
+      return {
+        exe = "~/projects/taxfyle-mobile/flutter/bin/dart",
```

Comment:
Should be in a standard path: `/opt/flutter/bin/dart` or when installed with [brew](https://dart.dev/get-dart) then it becomes `/opt/homebrew/bin/dart`

Reply:
Makes sens, just did it in that way to use the flutter version that I built using the flutter repository